osg-koji

```diff
@@ -264,11 +270,18 @@ def setup_koji_client_cert(user_cert, user_key):
             not os.path.samefile(KOJI_USER_CONFIG_DIR, OSG_KOJI_USER_CONFIG_DIR)) and
             os.path.isfile(OLD_CLIENT_CERT_FILE)):
 
-        if ask_yn("Copy client cert from '" + OLD_CLIENT_CERT_FILE + "' ?"):
+        if ask_yn("""
+You already have a client cert at '%s'.
```

Comment:
Don’t make me think!

Spell out words — research actually shows that doing so causes readers to think less (except when dealing with abbreviations and acronyms that are more common than their spelled-out versions, such as IBM and, arguably in our world, OSG).